query-engine/driver-adapters/src/queryable.rs

```diff
@@ -15,6 +15,17 @@ use quaint::{
 };
 use tracing::{info_span, Instrument};
 
+static SYSTEM_NAME: &'static str = if cfg!(feature = "mysql") {
+    "mysql"
+} else if cfg!(feature = "postgresql") {
+    "postgresql"
+} else if cfg!(feature = "sqlite") {
+    "sqlite"
+} else {
+    // compile_error!("No database feature enabled")
```

Comment:
The `compile_error!` directive is the one that makes the most sense here, but it doesn't work.

Reply:
Cargo features should be additive, could you please make it a method on `JsBaseQueryable` that will match on `self.provider` (or even a method on `ProviderFlavour` itself) instead?